config.json

```diff
@@ -209,7 +209,10 @@
       "unlocked_by": null,
       "difficulty": 1,
       "topics": [
-
+        "arrays",
```

Comment:
Could you please change it to `lists`. In my mind `arrays` store data of the same type, but lists can store data of any type (like in Python)

Reply:
Changed to `lists`